relays/substrate/src/millau_headers_to_rialto.rs

```diff
@@ -17,70 +17,52 @@
 //! Millau-to-Rialto headers sync entrypoint.
```

Comment:
```suggestion
//! Millau-to-Rialto finality sync entrypoint.
```

Although I wouldn't say that header sync is wrong either. It's just confusing during this transition time. What do you think?